src/solver/closure.go

```diff
@@ -19,8 +23,50 @@ func IsSupportedFunction(name string) bool {
 	return ok
 }
 
-// Struct containing information about the function that called the closure.
+// ClosureCallerInfo containing information about the function that called the closure.
 type ClosureCallerInfo struct {
 	Name     string          // caller function name
 	ArgTypes []meta.TypesMap // types for each arg for call caller function
 }
+
+func GetClosureName(fun *ir.ClosureExpr, curFunction, curFile string) string {
+	pos := ir.GetPosition(fun)
+	if curFunction != "" {
+		curFunction = "," + curFunction
+	}
+	name := `\Closure(` + curFile + curFunction + "):" + fmt.Sprint(pos.StartLine)
+	return name
+}
+
+func GetClosure(name ir.Node, sc *meta.Scope, cs *meta.ClassParseState) (meta.FuncInfo, bool) {
+	if !cs.Info.IsIndexingComplete() {
```

Comment:
I don't think `GetClosure` should be called during indexing at all.
All other `Get*` methods expect to be called during linting only.
If they're called during the indexing phase, it's probably a mistake.